dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -497,53 +498,62 @@ BlockInputStreamPtr Segment::getInputStreamForDataExport(const DMContext & dm_co
     return data_stream;
 }
 
+/// we will call getInputStreamRaw in two condition:
+///     1. Using 'selraw xxxx' statement, which is always in test for debug. (when filter_delete_mark = false)
+///        In this case, we will read all the data without mvcc filtering,
+///        del_mark = 1 filtering and sorted merge.
+///        We will just read all the data and return.
+///     2. We read in fast mode. (when filter_delete_mark = true)
+///        In this case, we will read all the data without mvcc filtering and sorted merge,
+///        but we will do del_mark = 1 filtering.
```

Comment:
```suggestion
/// we will call getInputStreamRaw in two condition:
///     1. Using 'selraw xxxx' statement, which is always in test for debug. (when filter_delete_mark = false)
///        In this case, we will read all the data without mvcc filtering,
///        del_mark != 0 filtering and sorted merge.
///        We will just read all the data and return.
///     2. We read in fast mode. (when filter_delete_mark = true)
///        In this case, we will read all the data without mvcc filtering and sorted merge,
///        but we will do del_mark != 0 filtering.
```
del_mark != 0 is more accurate, in case we use del_mark = 2 for different purpose in the future